src/gql/generated/graphcache.ts

```diff
@@ -1079,8 +1079,6 @@ export type Address = Node & {
   line1?: Maybe<Scalars['String']['output']>
   /** Second line of the address, if needed. Must be between 1 and 300 characters. */
   line2?: Maybe<Scalars['String']['output']>
-  /** The geographic location of the address. */
-  location?: Maybe<GeographyPoint>
```

Comment:
These lines should not be removed, they include a new feature. I think `maevsi/sqitch` was not on the most recent `master` when this files was regenerated.

Reply:
I've updated the schema